mne/surface.py

```diff
@@ -2228,3 +2228,181 @@ def _voxel_neighbors(seed, image, thresh=None, max_peak_dist=1,
                 break
         neighbors = next_neighbors  # start again checking all new neighbors
     return voxels
+
+
+class _Overlay(object):
```

Comment:
Rather than make this file even bigger, I think this could live in a `mne/viz/_3d_overlay.py` or so. It's really 3D viz specific, but I'd also rather not make `mne/viz/_3d.py` longer